Moviebuff/moviepage.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+ 
+struct details: View {
```

Comment:
Names for structs and classes should always be capitalized, so this would be `Details`. However, you should also use a descriptive name and it's convention to include something that indicates that this is a view. So maybe something like `MovieDetailView` would be appropriate.